includes/wikia/services/AvatarService.class.php

```diff
@@ -269,7 +269,8 @@ public static function getVignetteUrl( Masthead $masthead, $width, $timestamp )
 				$url = self::buildVignetteUrl( $width, $bucket, $relativePath, $timestamp, false );
 			}
 		} else { // default avatar
-			$legacyDefaultUrl = array_shift( $masthead->getDefaultAvatars( 'thumb/' ) );
+			$defaultAvatars = $masthead->getDefaultAvatars( 'thumb/' );
+			$legacyDefaultUrl = array_shift( $defaultAvatars );
```

Comment:
Or we can actually call `[0]` on the result of `getDefaultAvatars`. I'll change that tomorrow.